tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/FileHelper.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Azure.Sdk.Tools.Cli.Helpers
+{
+    /// <summary>
+    /// Helper class for file and directory operations.
+    /// </summary>
+    public static class FileHelper
+    {
+        /// <summary>
+        /// Validates that the specified directory exists and is empty.
+        /// </summary>
+        /// <param name="dir">The directory path to validate.</param>
+        /// <returns>An error message if validation fails, or null if validation passes.</returns>
+        public static string? ValidateEmptyDirectory(string dir)
+        {
+            if (string.IsNullOrWhiteSpace(dir))
```

Comment:
The error case being a non-empty string seems a bit off here. Do we want to throw an exception instead with a void return type, or is it not really an error case per se? If not, maybe better represented as an enum vs. a string so the caller doesn't have to parse the reason.